ironhide/lambdas/tin_parser/index.py

```diff
@@ -0,0 +1,30 @@
+"""
+TIN parser lambda
+"""
+
+import logging
+import traceback
+import json
+from typing import Dict, Any
+
+from ironhide.parsers.tin import TINParser
+
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+
+def handler(event: Dict[str, Any], lambda_context: Dict[str, Any]):
+    """
+    Handle events for TIN parsing.
+    """
+
+    message = json.loads(event["Records"][0]["body"])
```

Comment:
Event is used before checking if it has data in it. Add a check to see if event is nonempty before calling it.